### PR TITLE
Clarify existence of groups

### PR DESCRIPTION
**Pull request by KoBeWi**

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: doc/classes/SceneTree.xml [hunk 1]
```diff
@@ -137,6 +137,7 @@
 			<param index="0" name="name" type="StringName" />
 			<description>
 				Returns [code]true[/code] if the given group exists.
+				A group exists if any [Node] belongs to it (see [method Node.add_to_group]). Groups without nodes are removed automatically.
```
Comment (AThousandShips) [inline doc/classes/SceneTree.xml:140]:
```suggestion
				A group exists if any [Node] in the tree belongs to it (see [method Node.add_to_group]). Groups without nodes are removed automatically.
```
Nodes outside the tree can have groups, but don't count here, nitpick but might be clearer

Comment (AThousandShips) [reply doc/classes/SceneTree.xml:140]: Might even suggest merging the two lines into something like:
"Returns true if there any [Node]s in the tree belonging to this group..."

**Review by AThousandShips — APPROVED** [review]

**Comment by KoBeWi** [comment]
Global groups are editor-only, no? They don't exist in SceneTree.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**